packages/big-design/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -307,7 +307,7 @@ export const MultiSelect = typedMemo(
                   label: option.content,
                   onDelete: () => removeItem(option),
                 }))}
-                autoComplete="no"
+                autoComplete="off"
```

Comment:
I think this line was added by accident. Should be fine if we remove, can you test? Sorry about it!

Reply:
@jorgemoya Should I remove the autoComplete completely?

Reply:
This line, yes